lib/atomic-management.js

```diff
@@ -72,16 +75,54 @@ export default {
     });
   },
 
+
+  consumeStatusBar(statusBar) {
+    this.element = document.createElement('a');
+    this.statusBar = statusBar
+    return this.tile = this.statusBar.addLeftTile({item: this.element, priority: 100});
+  },
+
+  updateStatusBar() {
+    if(this.tooltips) {
+      this.tooltips.dispose()
+    }
+    if (this.tile) {
+      this.tile.destroy()
+    }
+    if (this.currentConfig && this.configuredFields) {
+      this.element = document.createElement('a');
+      this.element.classList.add(`icon-gear`)
+      this.element.classList.add('inline-block')
+      this.element.appendChild(document.createTextNode('-Local')); // Append the text to <button>
+      this.tooltips = atom.tooltips.add(this.element, {title: `Local Settings Configured in ${this.currentConfig}`})
+      function onClick(currentConfig) {
+        return function() {
```

Comment:
@josephch405 @dwy189 We can replace this with the emitter thing from PR 11 where we use a command that calls the function `openConfigFile), not actually doing it manually. Once it gets merged ill make that change this way I can just merge with master